src/main/java/team/gif/robot/UiSmartDashboard.java

```diff
@@ -1,6 +1,65 @@
 package team.gif.robot;
 
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import team.gif.lib.autoMode;
+import team.gif.lib.delay;
+import team.gif.robot.subsystems.drivers.Pigeon;
+
+import static team.gif.robot.Robot.*;
+import static team.gif.robot.Robot.drivetrain;
 
 public class UiSmartDashboard {
+
+    public SendableChooser<autoMode> autoModeChooser = new SendableChooser<>();
+    public SendableChooser<delay> delayChooser = new SendableChooser<>();
+
+    public UiSmartDashboard() {
+        ShuffleboardTab tab = Shuffleboard.getTab("SmartDashboard"); // Gets a reference to the shuffleboard tab
+        tab.add("BotHead", (x) -> {
+                    x.setSmartDashboardType("Gyro");
+                    x.addDoubleProperty("Value", () -> Robot.swervetrain.getHeading().getDegrees(), null);
```

Comment:
Why are we getting this from the drivetrain and not the pigeon?

Reply:
That's the function I wrote to get the pigeon heading and I didn't want to make the pigeon public